views/index.html

```diff
@@ -0,0 +1,38 @@
+{% block pageTitle %}
+Object to a company being struck off
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">
+            Object to a company being struck off
+        </h1>
+
+        <p>Use this service to tell us why a limited company should not be removed from the companies register.
+           You can only do this after the notice has been posted in The Gazette.</p>
+        <p>Companies House must receive the objection at least 2 weeks before the notice expiry date.</p>
+
+        <p>This service takes around 15 minutes.</p>
+
+        <a href="/sign-in" role="button" draggable="false" class="govuk-button govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8" data-module="govuk-button">
```

Comment:
The sign-in page is going to be achieved via an interceptor, so you'll just need to put the next page in here, not a link to sign-in

Reply:
Using link to company number page instead.